lab_6_pipeline/pipeline.py

```diff
@@ -144,17 +217,54 @@ def __init__(self, corpus_manager: CorpusManager):
         """
         Initializes MorphologicalAnalysisPipeline
         """
+        self._corpus = corpus_manager
 
     def _process(self, text: str) -> List[ConlluSentence]:
         """
         Returns the text representation as the list of ConlluSentence
         """
+        sentences = []
+        result = Mystem().analyze(re.sub(r'\W+', ' ', text))
```

Comment:
mystem instance should be created once and stored as own attribute in init

Reply:
Fixed